server/api/scheduler_test.go

```diff
@@ -107,8 +108,148 @@ func (s *testScheduleSuite) TestAPI(c *C) {
 		body, err := json.Marshal(input)
 		c.Assert(err, IsNil)
 		s.testAddAndRemoveScheduler(ca.name, ca.createdName, body, ca.extraTestFunc, c)
+		s.testPauseAndResumeScheduler(ca.name, ca.createdName, body, ca.extraTestFunc, c)
 	}
 
+	// test pause and resume all schedulers.
+
+	// add schedulers.
+	cases = cases[:3]
+	for _, ca := range cases {
+		input := make(map[string]interface{})
+		input["name"] = ca.name
+		for _, a := range ca.args {
+			input[a.opt] = a.value
+		}
+		body, err := json.Marshal(input)
+		c.Assert(err, IsNil)
+		s.addScheduler(ca.name, ca.createdName, body, ca.extraTestFunc, c)
+	}
+
+	// test pause all schedulers.
+	input := make(map[string]interface{})
+	input["delay"] = "5"
+	pauseArgs, err := json.Marshal(input)
+	c.Assert(err, IsNil)
+	err = postJSON(s.urlPrefix+"/all", pauseArgs)
+	c.Assert(err, IsNil)
+	time.Sleep(time.Second)
```

Comment:
Sleep too much makes the test slow, maybe you can reduce the sleep duration or use a more efficient way.

For example:
1. Set the delay to 600s to make sure you have enough time to check
2. Check if the schedulers are paused
3. Set the delay to 1s
4. sleep 1s and check if the scheduler  is resumed after the delay